tap_s3_csv/s3.py

```diff
@@ -178,29 +214,35 @@ def get_input_files_for_table(config, table_spec, modified_since=None):
 
         if (unmatched_files_count + matched_files_count) % max_files_before_log == 0:
             # Are we skipping greater than 50% of the files?
-            if 0.5 < (unmatched_files_count / (matched_files_count + unmatched_files_count)):
-                LOGGER.warn(("Found %s matching files and %s non-matching files. "
-                             "You should consider adding a `search_prefix` to the config "
-                             "or removing non-matching files from the bucket."),
-                            matched_files_count, unmatched_files_count)
+            if (unmatched_files_count / (matched_files_count + unmatched_files_count)) > 0.5:
+                LOGGER.warning(("Found %s matching files and %s non-matching files. "
+                                "You should consider adding a `search_prefix` to the config "
+                                "or removing non-matching files from the bucket."),
+                               matched_files_count, unmatched_files_count)
             else:
                 LOGGER.info("Found %s matching files and %s non-matching files",
                             matched_files_count, unmatched_files_count)
 
-    if 0 == matched_files_count:
+    if matched_files_count == 0:
         if prefix:
-            raise Exception('No files found in bucket "{}" that matches prefix "{}" and pattern "{}"'.format(bucket, prefix, pattern))
-        else:
-            raise Exception('No files found in bucket "{}" that matches pattern "{}"'.format(bucket, pattern))
+            raise Exception('No files found in bucket "{}" that matches prefix "{}" and pattern "{}"'
```

Comment:
Same, f-strings could be used here as in other repos, but up to you :)

Reply:
check my previous comment.